test/unit-tests/metafs/mim/metafs_io_scheduler_test.cpp

```diff
@@ -473,4 +473,43 @@ TEST_F(MetaFsIoSchedulerTexture, IssueRequest_testForProcessingRequestsForFilesW
     EXPECT_EQ(countArray[(size_t)MetaStorageType::NVRAM], 0);
     EXPECT_EQ(countArray[(size_t)MetaStorageType::JOURNAL_SSD], 0);
 }
+
+TEST_F(MetaFsIoSchedulerTexture, IssueRequest_testIf)
+{
+    // given
+    MetaFileExtent extents[2];
+    extents[0].SetStartLpn(0);
+    extents[0].SetCount(10);
+    extents[1].SetStartLpn(100);
+    extents[1].SetCount(10);
+    MetaFileContext fileCtx;
+    fileCtx.chunkSize = 4032;
+    fileCtx.fileBaseLpn = 0;
+    fileCtx.extentsCount = 2;
+    fileCtx.CopyExtentsFrom(extents, fileCtx.extentsCount);
+    MetaFsIoRequest originReq;
+    MetaFsIoRequest* req;
+
+    // when, 1st
+    req = new MetaFsIoRequest;
+    req->fileCtx = &fileCtx;
+    req->extents = req->fileCtx->extents;
+    req->extentsCount = req->fileCtx->extentsCount;
+    req->originalMsg = &originReq;
+    req->byteOffsetInFile = 4032 * 9;
+    req->byteSize = 4032;
+    req->targetMediaType = MetaStorageType::SSD;
+
+    // then
+    scheduler->IssueRequestAndDelete(req);
+    EXPECT_EQ(originReq.requestCount, 1);
+    EXPECT_EQ(metaIoWorker.GetRequestedCount(), 1);
+    EXPECT_EQ(metaIoWorker.GetTheFirstLpn(), 9);
+    EXPECT_EQ(metaIoWorker.GetTheLastLpn(), 9);
+    EXPECT_EQ(metaIoWorker.GetRequestedSize(), 4032);
+
+    // currentExtent_ have to be 0
```

Comment:
currentExtent_ have to be 0 because ... 와 같은 식으로 "because" 뒤에 뭔가 설명이 따라와야 자연스러울 것 같습니다 ㅎ 정석적으로는
```text
currentExtent_ has to be 0 because ... . Otherwise, ... 
```
이런 느낌으로 되면 좋을 것 같습니다 ㅎ

Reply:
수정하겠습니다 감사합니다